rollup.config.ts

```diff
@@ -34,7 +34,7 @@ const buildEsm = {
     'src/graphql.ts',
     'src/context/index.ts',
   ],
-  external: ['debug'],
+  external: ['debug', /@mswjs\/interceptors/],
```

Comment:
I think we should also add this regular expression to the `buildNode` and `buildNative` targets. They currently externalize `@mswjs/interceptors` and some specific interceptors, which will be matched as well since you've introducing a regular expression.